src/wp-includes/rest-api/endpoints/class-wp-rest-site-health-controller.php

```diff
@@ -336,7 +336,7 @@ protected function load_admin_textdomain() {
 		// Accounts for inner REST API requests in the admin.
 		if ( ! is_admin() ) {
 			$locale = determine_locale();
-			load_textdomain( 'default', WP_LANG_DIR . "/admin-$locale.mo" );
+			load_textdomain( 'default', WP_LANG_DIR . "/admin-$locale.mo", $locale );
```

Comment:
This one is redundant because `load_textdomain()` uses `determine_locale()` internally.

Reply:
The goal here is to avoid that second redundant determine_locale() call in load_textdomain()